tests/functional/parser/test_sqs.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+
+from aws_lambda_powertools.utilities.advanced_parser.envelopes.envelopes import Envelope
+from aws_lambda_powertools.utilities.advanced_parser.parser import parser
+from aws_lambda_powertools.utilities.typing import LambdaContext
+from tests.functional.parser.schemas import MyAdvancedSqsBusiness, MySqsBusiness
+from tests.functional.parser.utils import load_event
+from tests.functional.validator.conftest import sqs_event  # noqa: F401
+
+
+@parser(schema=str, envelope=Envelope.SQS)
```

Comment:
docs: could you add a comment as a context as to why someone would pass a schema as a str? should we expect something else?